ner4opt/constants.py

```diff
@@ -0,0 +1,107 @@
+class Constants:
+    """Class for all constants."""
+
+    MODELS_DIRECTORY = 'models'
+
+    LEXICAL = 'lexical'
+    LEXICAL_PLUS = 'lexical_plus'
+    SEMANTIC = 'semantic'
+    HYBRID = 'hybrid'
+
+    # CRF Models
+    LEXICAL_MODEL_NAME = 'lexical.pkl'
+    LEXICAL_PLUS_MODEL_NAME = 'lexical_plus.pkl'
+    HYBRID_MODEL_NAME = 'hybrid.pkl'
+
+    # Deep Models
+    # Roberta Model
+    SEMANTIC_DEEP_MODEL = 'skadio/ner4opt-roberta-v1'
+    # Roberta Model trained as part of our Hybridization experiment
+    HYBRID_DEEP_MODEL = 'skadio/ner4opt-roberta-v2'
+
+    # For generating these keywords automatically, please refer our jupyter notebooks
+    CONST_DIR_KEYWORDS = [
+        'not want to spend more than', 'not want to invest more than', 'can not sell more than',
+        'can not make more than', 'make only up to', 'can spend a total', 'can be used for', "n't make more than",
+        'not make more than', 'not be greater than', 'not eat more than', 'not prepare more than', 'no less than',
+        'can not exceed', 'at the most', 'maximum daily budget', 'does not exceed', 'has to collect', 'must not exceed',
+        'limited the number', 'limits the number', 'a total of', 'no more than', 'has on hand', 'at lest', 'more that',
+        'maximum budget', 'at most', 'not exceed', 'production capacity', 'in total', 'up to', 'minimum requirements',
+        'at least', 'can buy', 'available stock', 'more than', 'less than', 'available', 'inventory', 'capacity',
+        'more', 'acquired', 'has', 'invest', 'total', 'only', 'limit', 'limits', 'provide', 'have', 'competes',
+        'maximum', 'receives', 'budget', 'produces', 'using', 'requirements', 'below', 'in', 'at', 'stock', 'sells',
+        'minimum'
+    ]
+
+    OBJ_DIR_KEYWORDS = [
+        'minimizes', 'least', 'maximum', 'maximized', 'minimizing', 'using', 'lowest', 'maximise', 'fewest',
+        'overstate', 'most', 'minimize', 'high', 'maximal', 'minimal', 'minimum', 'maximize', 'minimise'
+    ]
+
+    SUBJECTS = ['nsubj', 'nsubjpass', 'csubj', 'csubjpass', 'agent', 'expl']
```

Comment:
add a comment where this is used

Reply:
OR maybe .. if these constants are all used for say Lexical.. Just add a BLOCK COMMENT for constants used in Lexical model add comment blokc #### at the top and bottom of it to keep them separate / distinguished